lua/which-key/text.lua

```diff
@@ -41,6 +43,10 @@ end
 function Text:set(row, col, str, group)
   str = str:gsub("[\n]", " ")
 
+  for i,v in ipairs(config.options.hidden) do 
```

Comment:
Great idea!

Would it be possible to do this change higher up the chain at this line: https://github.com/folke/which-key.nvim/blob/477d9d2846641be4bb0b51730bc2308648e17319/lua/which-key/keys.lua#L52

That's where all mappings are prepared for the layout. (I need to refactor it a bit though, it's getting a bit length 😅)

Your change should only run on mappings without label, to make sure we never change labels set by users.

Thanks!

Reply:
done in aada9b8